README.rst

```diff
@@ -148,6 +148,20 @@ The recipe supports the following options:
 **flake8**
     If set to True, run Flake8 code analysis. Default is ``True``.
 
+    Flake8 now takes advantage of ``flake8`` plugin system to cover some of
+    previously manually written linters. We've plugged in following extensions:
+
+    - ``flake8-blind-except``: Check for non correct try-except statements
+    - ``flake8-coding``: Check for utf8 headers
+    - ``flake8-debugger``: Check for ``pdb`` and ``ipdb`` statments
+    - ``flake8-quotes``: Check for double quotes
+    - ``pep8-naming``: Check methods and classes being aligned to PEP8 naming
+      conventions.
+
+    If you don't want to take advantage of some of the plugins, you may ignore
+    the messages using the ``flake8-ignore`` section below to ignore their
+    error codes.
```

Comment:
the only way to disable them from my point of view is to ship some default config for `flake8-ignore` otherwise we would have to dynamically load the extensions, imho this is currently not supported by the flake8 hook api.